### PR TITLE
Closes #893: increases margin between url and pocket

### LINKED ISSUE #893: Newly added tiles are cropped off-screen and can't be scrolled into view
Testing via master 0b63a732288395d2b8ec9cb9d8a63c01da650e64

When a new tile is added, it is cropped off-screen out of view and can't be scrolled into view

Probably a regression from the Pocket 'mega-tile' landing.

### PR DESCRIPTION
**Pull request by Baron-Severin**
cropped-tiles

### CODE REVIEW

**Comment by Baron-Severin** [comment]
@aminalhazwani Is this the behavior you were looking for?

tv-scrollview

**Comment by Baron-Severin** [comment]
@psymoon Passed UX.  

Changes made:
- Bottom margin was increased on id/navUrlInput
- Wrapped most of the file in a NestedScrollView
- Since I needed to increase indentation anyway (polluting the git history), I took the chance to run an automatic reformat on the file

File: app/src/main/res/layout/browser_overlay.xml [hunk 1]
```diff
@@ -1,92 +1,89 @@
...
+<org.mozilla.focus.browser.BrowserNavigationOverlayScrollView xmlns:android="http://schemas.android.com/apk/res/android"
     android:id="@+id/overlayScrollView"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:fillViewport="true"
     android:focusable="false"
     android:scrollbars="none">
 
-    <LinearLayout
+    <android.support.v4.widget.NestedScrollView
```
Comment (psymoon) [inline app/src/main/res/layout/browser_overlay.xml]: Is this for RecyclerView? I believe you can just set `app:layout_behavior="@string/appbar_scrolling_view_behavior"`

Comment (Baron-Severin) [reply app/src/main/res/layout/browser_overlay.xml]: Unfortunately, we get some weird scrolling behavior with that approach.

Comment (psymoon) [reply app/src/main/res/layout/browser_overlay.xml]: Alas, then looks good!

**Pull request merged**